doc/manual/manual.tex

```diff
@@ -1756,6 +1756,127 @@ \subsection{Compiling a static \aspect{} executable}
 
 The here mentioned build was tested on a Cray XC30 cluster, which was set up for default static compiling and linking. On machines that default to dynamic linking additional compiler and/or linker flags may be required (e.g. "-fPIC" / "--static"). In case of questions send an email to the mailing list.
 
+
+\subsection{Installing and running \aspect{} on Mac OS X}
+%\author{Hiroaki Matsui \textless\href{hrmatsui@ucdavis.edu}{hrmatsui@ucdavis.edu}\textgreater}
+
+Mac OS X has some eccentricities which can complicate installation of \aspect{}. Currently the easiest and most reliable way to run \aspect{} under Mac OS X Mavericks (10.9) and Yosemite (10.10) is to install and run the binary package for \dealii{}. The step-by-step process is described in detail, with screenshots, here: \url{https://wiki.geodynamics.org/_media/software:aspect:aspect_yosemite_20150529.pdf}
+
+\begin{enumerate}
+\item Install Cmake.
+
+CMake is a cross-platform, open-source build system that can be downloaded from \url{http://www.cmake.org}.  After installation of CMake.app, the terminal command for cmake will be
+\begin{verbatim}
+% /Applications/CMake.app/Contents/bin/cmake
```

Comment:
They will show up because of the verbatim, but I will change it for
consistency with later notation.

On Fri, May 29, 2015 at 4:40 PM, Timo Heister <notifications@github.com>
wrote:

> In doc/manual/manual.tex
> <https://github.com/geodynamics/aspect/pull/496#discussion_r31373572>:
>
> > @@ -1756,6 +1756,127 @@ \subsection{Compiling a static \aspect{} executable}
> >
> >  The here mentioned build was tested on a Cray XC30 cluster, which was set up for default static compiling and linking. On machines that default to dynamic linking additional compiler and/or linker flags may be required (e.g. "-fPIC" / "--static"). In case of questions send an email to the mailing list.
> >
> > +
> > +\subsection{Installing and running \aspect{} on Mac OS X}
> > +%\author{Hiroaki Matsui \textless\href{hrmatsui@ucdavis.edu}{hrmatsui@ucdavis.edu}\textgreater}
> > +
> > +Mac OS X has some eccentricities which can complicate installation of \aspect{}. Currently the easiest and most reliable way to run \aspect{} under Mac OS X Mavericks (10.9) and Yosemite (10.10) is to install and run the binary package for \dealii{}. The step-by-step process is described in detail, with screenshots, here: \url{https://wiki.geodynamics.org/_media/software:aspect:aspect_yosemite_20150529.pdf}
> > +
> > +\begin{enumerate}
> > +\item Install Cmake.
> > +
> > +CMake is a cross-platform, open-source build system that can be downloaded from \url{http://www.cmake.org}.  After installation of CMake.app, the terminal command for cmake will be
> > +\begin{verbatim}
> > +% /Applications/CMake.app/Contents/bin/cmake
>
> or do they because this is inside verbatim? not sure
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/aspect/pull/496/files#r31373572>.
>